client/my-sites/media-library/index.jsx

```diff
@@ -1,73 +1,61 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classNames = require( 'classnames' ),
-	isEqual = require( 'lodash/isEqual' );
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import isEqual from 'lodash/isEqual';
```

Comment:
cannot find the original source, but I believe `import module from lodash/module` is being deemphasized in favor of  `import { map as _map } from 'lodash';`